data/recipes.yaml

```diff
@@ -486,10 +486,19 @@
   required:
   - [1, furnace]
 
+- in:
+  - [1, gold]
+  out:
+  - [8, gold coin]
+  required:
+  - [1, big furnace]
+
 - in:
   - [1, gold coin]
   - [1, bit (1)]
   - [1, bit (0)]
+  required:
+  - [1, metal drill]
```

Comment:
Why is a metal drill required to make a bitcoin?

Reply:
Because you are mining bitcoin?

Reply:
> Because you are mining bitcoin?

Ohhhhh of course. :smile:

Reply:
> It does not really matter, you already have a drill.

Well, the robot that mined the gold has a drill.  But maybe you have a different robot programmed to mine the bitcoins.  But it's fine, you can give that robot a drill too.